app/models/course_reserve.rb

```diff
@@ -1,35 +1,8 @@
 # frozen_string_literal: true
 
 # The store of course reserve data from courses.json
```

Comment:
I think this comment is no longer accurate.

Reply:
I changed it